dev/sass/sample.scss

```
@@ -27,6 +27,31 @@ body {
   font-family: 'Aboreto', cursive;
   background: $blackColor;
 }
+// Nav
+nav{
```

```suggestion
nav.navigation{
```
It's a risky to style elements only by the tag name (you can have multiple `nav` elements on the site).

Instead of that you should always use class names (wich you have right now inside HTML code) or even mix of tag-name & class name

---

Thanks. 
But when I am adding nav.navigation my menu section is no longer formatted (for unknown reason for me).
I leave at this moment just .navigation